tee-worker/litentry/core/credentials/src/oneblock/mod.rs

```diff
@@ -20,11 +20,11 @@ impl OneBlockAssertionUpdate for Credential {
 	fn update_notion_assertion(&mut self, course_type: &OneBlockCourseType, value: bool) {
 		let (assertion_content, info) = match course_type {
 			OneBlockCourseType::CourseCompletion =>
-				("is_course_completion", VC_ONEBLOCK_COURSE_INFOS[0]),
-			OneBlockCourseType::CourseExcellenceCompletion =>
-				("is_course_excellence_completion", VC_ONEBLOCK_COURSE_INFOS[1]),
+				("$is_course_completion", VC_ONEBLOCK_COURSE_INFOS[0]),
```

Comment:
can you double the column "Assertion Breakdown (field name/value)" in the table? I think they are still mismatched.

Reply:
You mean `OneBlockCourseType` and `COURSE_INFOS` mismatched?

Reply:
Thanks @BillyWooo, the description on the document is also inconsistent, and I am currently waiting for Kayla's confirmation